docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,953 @@
+     padding: 20px;
+     margin-top: 10px;
+     max-width: 320px;
+     font-size: large;
+   }
+
+   .key {
+     max-width: 200px;
+     padding: 0px 5px 5px;
+   }
+
+   .key p {
+     padding: 4px;
+     margin: 0px;
+   }
+
+   .good {
+     background-color: lightgreen;
+   }
+
+   .moderate {
+     background-color: yellow;
+   }
+
+   .unhealthy-sensitive {
+     background-color: orange;
+   }
+
+   .unhealthy {
+     background-color: red;
+   }
+
+   .very-unhealthy {
+     background-color: violet;
+   }
+
+   .hazardous {
+     color: white;
+     background-color: purple;
+   }
+
+   #main {
+     max-width:600px;
+     padding:10px 30px 10px;
+   }
+   ```
+
+   Feel free to adjust any of the colors, margins, fonts, and other specifications in <file>style.css</file> based on your preferences.
+
+## Run the code
+
+1. In a command prompt terminal, navigate to your <file>aqi-dashboard</file> directory.
+   Run the following command to start up your air quality dashboard:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   npm start
+   ```
+
+   {{<imgproc src="/tutorials/air-quality-fleet/terminal-url.png" resize="x900" declaredimensions=true alt="Terminal window with the command 'npm start' run inside the aqi-dashboard folder. The output says 'start' and then 'esbuild' followed by the esbuild string from the package.json file you configured. Then there's 'Local:' followed by a URL and 'Network:' follwed by a different URL." style="max-width:700px" >}}
+
+1. The terminal should output a line such as `Local:  http://127.0.0.1:8000/`.
+   Copy the URL the terminal displays and paste it into the address bar in your web browser.
+   
+   {{% alert title="Tip" color="tip" %}}
+To view the dashboard from the browser of a different device on the same network, you can do so using the `Network:` URL.
+{{% /alert %}}
+
+   The data may take up to approximately 5 seconds to load, then you should see air quality data from all of your sensors.
+   If the dashboard does not appear, right-click the page, select **Inspect**, and check for errors in the console.
+
+   {{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/three-sensor-dash.png" resize="x900" declaredimensions=true alt="Air quality dashboard in a web browser with PM2.5 readings from three different sensor machines displayed." style="max-width:500px" >}}
+
+   Great work.
+   You've learned how to configure a fleet of machines, sync their data to one place, and pull that data into a custom dashboard using TypeScript.
+
+## Next steps
+
+Now that you can monitor your air quality, you can try to improve it and see if your efforts are effective.
+You might try putting an air filter in your home or office and comparing the air quality data before you start running the filter with air quality after you have run the filter for a while.
+Or, try sealing gaps around doors, and check whether your seal is working by looking at your dashboard.
```

Comment:
I think this needs to be reframed a bit. How about

```suggestion
Now you can monitor air quality across as many machines as you have.
If you want to improve the air quality in a specific location, you might try putting an air filter in your home or office. Or, you could try sealing gaps around doors, and check whether your seal is working by looking at your dashboard.
With the dashboard, you will be able to see whether the results of your efforts are effective.
You could also improve the dashboard to show your air quality at different time points to have an even better overview.
```

Still keeps the content but I think it fits better

Reply:
```suggestion
Now you can monitor air quality across your entire fleet of sensing machines.
	If you want to improve the air quality in a specific location, you might try putting an air filter in your home or office.
	Or, you could try sealing gaps around doors.
	With the dashboard, you will be able to see whether the results of your efforts are effective.
	You could also improve the dashboard to show your air quality at different time points to have an even better overview.
```
Okay sure. Modifying slightly for redundancy reduction and because "as you have" doesn't feel natural to me.